Dockerfile

```diff
@@ -83,10 +83,7 @@ RUN go get -u github.com/coinbase/protoc-gen-rbi
 RUN go get github.com/gomatic/renderizer/cmd/renderizer
 
 # Origin protoc-gen-go should be installed last, for not been overwritten by any other binaries(see #210)
-RUN set -e && \
-  GO111MODULE=on go get -u github.com/golang/protobuf/protoc-gen-go@v1.3.5 && \
-  cd /go/pkg/mod/github.com/golang/protobuf@v1.3.5/protoc-gen-go && \
-  go install .
+RUN go get -u github.com/golang/protobuf/protoc-gen-go
```

Comment:
Why do we no longer need to run `go install .`?

Reply:
`go get -u` in GOPATH mode is doing `go install` under the hood, but it's not supporting `path@version` so had to do that workaround in the #211